firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -318,6 +322,11 @@ private void sendScreenTrace(Activity activity) {
      * FrameMetricsAggregator#remove(Activity)} will throw exceptions for hardware acceleration
      * disabled activities.
      */
+    try {
+      frameMetricsAggregator.remove(activity);
```

Comment:
What will happen if you do not remove the activity and only reset?

Reply:
The way FrameMetricsAggregator works is by [registering a listener](https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/view/Window.java;l=928;drc=master;bpv=1;bpt=1) to the DecorView of a `Window` when you call `FrameMetricAggregator.add(activity)`. Purely for consistency sake I think it is important to remove the listener when we don't want to monitor that activity anymore.

Reply:
Sounds good for consistency